eap74-rhel8-payg-vmss/src/main/scripts/jbosseap-setup-redhat.sh

```diff
@@ -4,7 +4,12 @@ log() {
         printf '%s %s\n' "$(date "+%Y-%m-%d %H:%M:%S")" "$line" >> /var/log/jbosseap.install.log
     done
 }
-
+# firewalld installation and configuration
+if [[ "${JDK_VERSION,,}" != "openjdk8" ]]; then
+    sudo yum install firewalld -y
+    sudo systemctl start firewalld
+    sudo systemctl enable firewalld
+fi
```

Comment:
Can you assure me that this will not cause the whole thing to fail if `firewalld` is already installed on the VM? If not, please revise so that the whole thing will not fail if `firewalld` is already installed on the system.

Reply:
I have added this, this will only run it is not installed. 

https://github.com/Azure/rhel-jboss-templates/pull/213/commits/ce84b7c4d7efae75279b4be61f2a3b50455c6894